doctest/build.gradle

```diff
@@ -48,6 +48,9 @@ doctest.finalizedBy stopOpenSearch
 build.dependsOn doctest
 clean.dependsOn(cleanBootstrap)
 
+String mlCommonsRemoteFile = 'https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.0.0-alpha1/latest/linux/x64/builds/opensearch/plugins/opensearch-ml-2.0.0.0-alpha1.zip'
```

Comment:
sql is on rc1, use this instead? https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.0.0-rc1/2500/linux/x64/tar/builds/opensearch/plugins/opensearch-ml-2.0.0.0-rc1.zip

also is it possible to substitute version with variable?

Reply:
something like https://github.com/opensearch-project/cross-cluster-replication/pull/366

Reply:
updated it in the latest revision

Reply:
Will update it to remove zip dependency